prov/verbs/src/fi_verbs.h

```diff
@@ -357,9 +372,11 @@ struct verbs_ep_domain {
 extern const struct verbs_ep_domain verbs_rdm_domain;
 
 int fi_ibv_check_ep_attr(const struct fi_ep_attr *attr,
-			 const struct fi_info *info);
+			 const struct fi_info *info,
+			 int srx_ctx_support);
```

Comment:
Can't the domain attribute max_ep_srx_ctx be set/used to see if SRQs are supported and can be used?

Reply:
Sure, we can. Thanks.